examples/source-restic/source-restic.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# This shows an example of how the mover container can be invoked, though it is
+# not intended to be run manually (only by the operator).
+
+
+# This Secret is provided by the user to configure the restic repo parameters.
+# The keys will be turned into env vars
+apiVersion: v1
+kind: Secret
+metadata:
+  name: restic-config
+type: Opaque
+stringData:
+  # The repository url
+  RESTIC_REPOSITORY: s3:http://minio.minio.svc.cluster.local:9000/restic-repo
+  # The repository encryption key
+  RESTIC_PASSWORD: my-secure-restic-password
+  # ENV vars specific to the back end
+  # https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html
+  AWS_ACCESS_KEY_ID: access
+  AWS_SECRET_ACCESS_KEY: password
+
+
+---
+# This volume persists the restic cache between runs. The data is truly a cache
+# (and can be deleted), but it speeds subsequent runs by decreasing the amount
+# of data that must be fetched from the repo.
+kind: PersistentVolumeClaim
```

Comment:
Should this be in the example? The operator creates it, right?

Reply:
The operator will create it. It's dangling code.